nixos/modules/security/acme/default.nix

```diff
@@ -142,43 +120,72 @@ let
           chown -R ${user}:${data.group} "$fixpath"
         fi
       done
-    '') certConfigs));
-  in {
-    description = "Fix owner and group of all ACME certificates";
+    '') certConfigs))
+  );
 
+  # This is defined with lib.mkMerge so that we can separate the config per function.
+  setupService = lib.mkMerge [
+  {
```

Comment:
The content of the list should all be indented one more layer. There's already enough changed lines it's worth making it pretty IMO.

Reply:
Eh, yeah I did want to preserve the git blame but you're right, hardly any of it remains.